mitmproxy/addons/share.py

```diff
@@ -27,12 +28,6 @@ def open_file(self, path):
         path = os.path.expanduser(path)
         return open(path, mode)
 
-    def read_file(self, path):
-        path = os.path.expanduser(path)
-        with open(path, "rb") as f:
-            content = f.read()
-        return content
-
     def start_stream_to_path(self, path, flt):
```

Comment:
I need to look into these methods more before removing them. The next commit will be without all the unnecessary methods.